### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by CouchersBot**
Translations update from [Weblate](https://translate.couchershq.org/projects/couchers/web-app-donations/)
for Couchers/Web app - Donations.

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by darrenvong — CHANGES_REQUESTED** [review]
Don't merge until this has been proofread and approved first while we're still ironing the Weblate workflow

**Review by aapeliv — APPROVED** [review]

**Pull request merged**
